src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +44,21 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
-createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#if (!MIN_VERSION_http_conduit(2,1,11))
+  where
+    parseUrlThrow = parseUrl
+#endif
 
 getResponse :: OAuth -> Credential -> Request -> IO (Response BS.ByteString)
-getResponse oauth credential req =
-  withManager $ \m -> do
-    signedreq <- signOAuth oauth credential req
-    httpLbs signedreq m
+getResponse oauth credential req = do
+  m <- newManager tlsManagerSettings
+  signedreq <- signOAuth oauth credential req
+  httpLbs signedreq m
+#if (!MIN_VERSION_http_conduit(2,1,11))
+    where
+      tlsManagerSettings = defaultManagerSettings
```

Comment:
@CristhianMotoche CPP is fine here, but I'm not sure defaultManagerSettings and tlsManagerSettings are doing the same thing. The solution could still be CPP if it's simpler.

http-client-tls is installed anyway because http-conduit depends on it, so not much would actually change in terms of dependencies.